py/surveyqa/nightly.py

```diff
@@ -241,6 +241,17 @@ def makeplots(night, exposures, tiles, outdir):
 
     Writes outdir/night-*.html
     '''
+<<<<<<< HEAD
+    #getting path for the previous and next night links, first and last night links, link back to summary page
+=======
+    
+    #getting path for the previous and next night links
+>>>>>>> 2740d584a08dd6f2e9054ea0c4833525bd611a9e
+    [prev_str, next_str] = get_night_link(night, exposures)
+    first_str = get_night_link(exposures['NIGHT'][0], exposures)[0]
+    last_str = get_night_link(exposures['NIGHT'][-1], exposures)[1]
+    summary_str = "file://" + str(outdir) + "/summary.html"
```

Comment:
These references to `file://` in a specific directory will only work for local browsing, not when these files are served by a webserver.  I think what you need here is just
```python
summary_str = "summary.html"
```
i.e. rather than referring to a `file://` in a specific directory, just give the filename and it will be discovered relative to the location of the file containing the link.